core/lib/types/src/protocol_version.rs

```diff
@@ -70,6 +73,7 @@ impl ProtocolVersionId {
             ProtocolVersionId::Version15 => VmVersion::VmVirtualBlocks,
             ProtocolVersionId::Version16 => VmVersion::VmVirtualBlocksRefundsEnhancement,
             ProtocolVersionId::Version17 => VmVersion::VmVirtualBlocksRefundsEnhancement,
+            ProtocolVersionId::Version18 => VmVersion::VmVirtualBlocksRefundsEnhancement,
```

Comment:
nit: shall we have `todo` here too?

Reply:
we could, originally @perekopskiy said we should use what v17 was using but if its going to be changed with the overall boojum integration merge i think todo is safe

Reply:
we can't use todo because of upgrade test..